admin-center/display/system-alert.md

```diff
@@ -1,8 +1,13 @@
 ---
-layout: post
+layout: page
 title: Adding a Custom System Alert to Your Results Page
 category: manual
 tags: help-manual manage-display alert
+date: February 7, 2018
+last_modified_at: February 7, 2018
+sidenav: admin_center
+redirect_from:
+    - /manual/system-alert.html
 ---
 
 [Search.gov Home]({{ site.baseurl }}/index.html) > [Admin Center](https://search.usa.gov/sites/) > YourSite > Display > Search Page Alert
```

Comment:
I missed this one. Please prepend this line with `Find it in the Admin Center: `